package.json

```diff
@@ -5,20 +5,24 @@
   "description": "Horondi",
   "main": "server.js",
   "repository": {
+    "type": "git",
     "url": "https://github.com/horondi/horondi_client_be.git"
   },
   "private": "true",
   "scripts": {
     "start": "nodemon server.js",
     "lint": "eslint .",
-    "lint:fix": "eslint . --fix"
+    "lint:fix": "eslint . --fix",
+    "test": "jest --coverage"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@azure/identity": "^1.0.3",
+    "@azure/keyvault-secrets": "^4.0.4",
     "@hapi/joi": "^17.1.1",
     "apollo-server": "^2.14.5",
-    "bcrypt": "^5.0.0",
```

Comment:
Why have you removed `bcrypt` dependency?

Reply:
bcrypt was added at the beggining. but now we use bcryptjs. so this bcrypt  is not used any more